### DISCUSSION TITLE
How to use create Tool Page with Vertical Navigation?

### DISCUSSION THREAD

**Discussion by stevschm** [UI5/webcomponents#8563] [Q&A] [answered]
Hi,
I search the docs, but somehow couldn't the needed information.
My question is, how can I create a Tool Page with Vertical Navigation like here: https://sapui5.hana.ondemand.com/#/entity/sap.tnt.ToolPage/sample/sap.tnt.sample.ToolPage.

I only found the ShellBar from the Fiori category. I also saw that there will be some styling. But could not find any information.

Best regards,
Steve

**Comment by petyabegovska** [comment, marked as answer]
Hi @stevschm,

Currently, we provide this sample: https://sap.github.io/ui5-webcomponents/nightly/components/fiori/SideNavigation/#sidenavigation-in-tool-layout, which has very close design to the sap.tnt Tool Page. In future, we are planing to provide easier consumption with reusable component styles or new component.

Best regards,
Petya

**Reply by stevschm** [reply]
Perfect! Thank you!
